tests/optimizer/test_scheduler.py

```diff
@@ -0,0 +1,172 @@
+"""
+Unit tests for the zeus.optimizer.pipeline_frequency.server.scheduler module
+"""
```

Comment:
Please fix docstring style across all files following those documented in the Google style guide: https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings